CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ if(WIN32)
         set(CMAKE_INSTALL_LIBDIR ".")
         set(CMAKE_INSTALL_DATADIR ".")
 else()
+	set(TFE_ICONDIR "/usr/local/share/icons/hicolor")
```

Comment:
Don't set absolute paths in here, this will break if you do a "make DESTDIR=$PWD/image install".
Instead, above line 37, do something like "set(TFE_ICONDIR "share/icons/hicolor")", same for desktopdir.